src/site/content/en/blog/appcache-removal/index.md

```diff
@@ -74,7 +73,7 @@ A "deprecated" feature still exists, but logs warning messages discouraging use.
     <tr>
     <td>Complete removal from secure contexts for everyone, with completion of origin trial
     </td>
-    <td>Chrome 93 (<a href="https://chromiumdash.appspot.com/schedule">estimated October 2021</a>)
+    <td>Chrome 95 (<a href="https://chromiumdash.appspot.com/schedule">estimated October 2021</a>)
```

Comment:
```suggestion
    <td>Chrome 95 (<a href="https://chromiumdash.appspot.com/schedule">October 5th, 2021</a>)
```

I think it's safe to say October 5th, as that's the end date of the origin trial.  (Looks good otherwise!)

Reply:
Thanks! I've also removed the "(estimated)" from the other dates in the past.